AEPIdentity/Sources/IdentityState.swift

```diff
@@ -466,9 +467,9 @@ class IdentityState {
     /// Generates the ecid if not cached and save it to persistence
     private func generateAndPersistECID() {
         if identityProperties.ecid == nil {
-            Log.trace(label: "\(LOG_TAG):\(#function)", "Generating new ECID, ECID not found in persistence.")
             identityProperties.ecid = ECID()
             identityProperties.saveToPersistence()
+            Log.trace(label: "\(LOG_TAG):\(#function)", "Generating new ECID value \(identityProperties.ecid?.ecidString ?? "nil")")
```

Comment:
can we also add traces on the flow that triggered this call to generateAndPersistECID?